BB-SC/MWLayer/Communication/BB-SC-Constraint-DDS-embeddedRTPS.md

```diff
@@ -0,0 +1,94 @@
+# embeddedRTPS - DDS for Ressource-constrained environments
+
+## BB Tags(s)
+
+- SC, CSC, CS-MU, CSC-MU

```

Comment:
Use full name of BB tag. eg. BB-SC so that it is in line with the README file and the other BBs

Reply:
Added prefix BB-* to all Tag(s)